src/dataflow/src/logging/materialized.rs

```diff
@@ -315,7 +315,7 @@ pub fn construct<A: Allocate>(
             move |((consumer_name, source_id, partition_id), diff_vector)| {
                 row_packer.pack(&[
                     Datum::String(&consumer_name),
-                    Datum::String(&source_id.to_string()),
+                    Datum::String(&source_id.source_id.to_string()),
```

Comment:
I'm not sure if this is correct now, I think that you need both source_id and dataflow_id to uniquely identify these metrics?

Reply:
Oh, you're right! I just pushed a fix for this.